contrib/download-frozen-image-v2.sh

```diff
@@ -47,6 +47,37 @@ registryBase='https://registry-1.docker.io'
 authBase='https://auth.docker.io'
 authService='registry.docker.io'
 
+ARM_VARIANT=""
+
+fetch_arm_variant() {
+	if [ "$(go env GOHOSTOS)" = "linux" ];then
+		local cpuinfo="$(cat /proc/cpuinfo | grep -i 'CPU architecture' | head -n 1 -)"
+		local variant="$(echo -e "${cpuinfo##*:}" | tr -d '[:space:]')"
```

Comment:
So this PR tries to pull the image with the OSType/Arch abstracted from the host which is running the code here, while not  to specify a different Arch than the local host.